networkdb/cluster.go

```diff
@@ -107,7 +107,6 @@ func (nDB *NetworkDB) clusterInit() error {
 		config.BindPort = nDB.config.BindPort
 	}
 
-	config.ProtocolVersion = memberlist.ProtocolVersionMax
```

Comment:
Can you set it to `ProtocolVersion2Compatible` explicitly ? Letting `DefaultLANConfig` pick a version `under the rug` will break us if that version changes in the next memberlist bump.